bindings/rethinkdb/statechange/statechange.go

```diff
@@ -0,0 +1,187 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package statechange
+
+import (
+	"encoding/json"
+	"strconv"
+	"strings"
+	"time"
+
+	r "github.com/dancannon/gorethink"
+	"github.com/dapr/components-contrib/bindings"
+	"github.com/dapr/dapr/pkg/logger"
+	"github.com/pkg/errors"
+)
+
+// Binding represents RethinkDB change change state input binding which fires handler with
+// both the previous and current state store content each time there is a change.
+type Binding struct {
```

Comment:
I think the name Binding is too generic since the struct is also exported.

Reply:
It is in a Rethink package so this is a good way to prevent stuttering `rethinkdb.RethinkDB`. `rethinkdb.Binding` seems cleaner, but I realize this is the convention we've been following so I'll fix this once it lands.

Reply:
To be honest, each component has the different convention :)  In Go convention, Binding is recommended. But the best convention is following the existing one.